standalone_app/package.json

```diff
@@ -1,37 +1,41 @@
 {
-  "name": "optuna-dashboard-wasm",
-  "private": true,
-  "version": "0.0.0",
-  "description": "",
-  "scripts": {
-    "watch": "vite",
-    "build:vscode": "webpack"
-  },
-  "devDependencies": {
-    "@types/plotly.js": "^2.29.2",
-    "@types/react": "^18.2.64",
-    "@types/react-dom": "^18.2.21",
-    "@types/react-router-dom": "^5.3.3",
-    "@vitejs/plugin-react": "^4.2.1",
-    "compression-webpack-plugin": "^11.1.0",
-    "esbuild-loader": "^4.0.3",
-    "vite": "^5.1.5",
-    "webpack": "^5.90.3",
-    "webpack-cli": "^5.1.4"
-  },
-  "dependencies": {
-    "@emotion/react": "^11.11.3",
-    "@emotion/styled": "^11.11.0",
-    "@mui/icons-material": "^5.15.12",
-    "@mui/lab": "^5.0.0-alpha.167",
-    "@mui/material": "^5.15.12",
-    "@optuna/storage": "../tslib/storage",
-    "module-workers-polyfill": "^0.3.2",
-    "notistack": "^3.0.1",
-    "optuna": "../rustlib/pkg",
-    "plotly.js-dist-min": "^2.30.0",
-    "react": "^18.2.0",
-    "react-dom": "^18.2.0",
-    "react-router-dom": "^6.22.3"
-  }
+	"name": "optuna-dashboard-wasm",
+	"private": true,
+	"version": "0.0.0",
+	"description": "",
+	"scripts": {
+		"watch": "vite",
+		"build:vscode": "webpack"
+	},
+	"devDependencies": {
+		"@types/plotly.js": "^2.29.2",
+		"@types/react": "^18.2.64",
+		"@types/react-dom": "^18.2.21",
+		"@types/react-router-dom": "^5.3.3",
+		"@vitejs/plugin-react": "^4.2.1",
+		"compression-webpack-plugin": "^11.1.0",
+		"esbuild-loader": "^4.0.3",
+		"vite": "^5.1.5",
+		"webpack": "^5.90.3",
+		"webpack-cli": "^5.1.4"
+	},
+	"dependencies": {
+		"@emotion/react": "^11.11.3",
+		"@emotion/styled": "^11.11.0",
+		"@mui/icons-material": "^5.15.12",
+		"@mui/lab": "^5.0.0-alpha.167",
+		"@mui/material": "^5.15.12",
+		"@optuna/react": "../tslib/react",
+		"@optuna/storage": "../tslib/storage",
+		"module-workers-polyfill": "^0.3.2",
+		"notistack": "^3.0.1",
+		"optuna": "../rustlib/pkg",
+		"plotly.js-dist-min": "^2.30.0",
+		"react": "^18.2.0",
+		"react-dom": "^18.2.0",
+		"react-router-dom": "^6.22.3"
+	},
+	"optionalDependencies": {
+		"@rollup/rollup-linux-x64-gnu": "^4.13.2"
+	}
```

Comment:
Memo:These lines are added to resolve the error due to rollup dependency (module not found error: `Error: Cannot find module @rollup/rollup-linux-x64-gnu`)

ref: https://stackoverflow.com/questions/77569907/error-in-react-vite-project-due-to-rollup-dependency-module-not-found